mpilib.f90

```diff
@@ -8,34 +8,41 @@ module mpilib_mod
 
   !---END USE
 
+  !YuP[2019-05-31] These subroutines could be moved into mpins_par.f (in /mpi/)
```

Comment:
sounds good, less is more. note that i made a note to make your note into an issue.

Reply:
see #18